WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+JetpackInstall.swift

```diff
@@ -2,34 +2,66 @@ import UIKit
 
 extension MySiteViewController {
     func subscribeToJetpackInstallNotifications() {
-        NotificationCenter.default.addObserver(self, selector: #selector(launchJetpackInstallFromNotification(_:)), name: .installJetpack, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(promptJetpackPluginInstallFromNotification(_:)), name: .promptInstallJetpack, object: nil)
     }
 
-    /// Presents the Jetpack Login (Install) controller from a notification
-    @objc func launchJetpackInstallFromNotification(_ notification: NSNotification) {
-        guard let blog = blog else {
+    /// Presents the prompt to install Jetpack from a notification
+    @objc func promptJetpackPluginInstallFromNotification(_ notification: NSNotification) {
+        guard let blog = blog,
+            let promptSettings = notification.userInfo?["jetpackInstallPromptSettings"] as? JetpackInstallPromptSettings else {
             return
         }
 
-        let controller = JetpackLoginViewController(blog: blog)
-        controller.promptType = .installPrompt
+        let installPromptViewController = JetpackInstallPromptViewController(blog: blog)
+        let installPromptNavigationController = UINavigationController(rootViewController: installPromptViewController)
```

Comment:
It looks like this prompt screen is "standalone" because the installation flow is another modal on top of it (it's not pushed onto the nav controller created on the line above). If it's "standalone", is the nav controller needed?

Reply:
Good observation 👍. In the current situation, it's currently not needed, since we don't have any navigation flow inside the prompt.